main.js

```diff
@@ -129,7 +140,7 @@ function timeAgo (date, interval) {
 	}
 }
 
-var init = function(el) {
+ODate.init = function(el) {
```

Comment:
If `init` was an instance method it would require `ODate` to be instantiated before being used. If we attach the method to the constructor function it acts like a static method and can be used without instantiation of the class/blueprint.